app/src/main/java/fr/tvbarthel/ivanbnw/player/TrackView.java

```diff
@@ -63,6 +73,13 @@ public TrackView(Context context, AttributeSet attrs, int defStyleAttr) {
      */
     public void presentData(@NonNull SoundCloudTrack track) {
         trackTitle.setText(track.getTitle());
+        long seconds = track.getDurationInMilli() / 1000;
+        duration.setText(String.format(durationFormat, seconds / 60, seconds % 60));
```

Comment:
Just to keep track of our IRL discussion: OK to keep the String format for now =)